modules/cluster-api/graph/server.go

```diff
@@ -16,9 +16,9 @@ package graph
 
 import (
 	"context"
-	"errors"
+	"fmt"
+	sharedWebsocket "github.com/kubetail-org/kubetail/modules/shared/graphql/websocket"
```

Comment:
The rest of the codebase places internal imports in a group at the bottom and we use all lowercase names:
```
...
"time"

"github.com/99designs/gqlgen/graphql/handler"
...
"github.com/vektah/gqlparser/v2/ast"

grpcdispatcher "github.com/kubetail-org/grpc-dispatcher-go"

"github.com/kubetail-org/kubetail/modules/shared/graphql/directives"
"github.com/kubetail-org/kubetail/modules/shared/k8shelpers"
sharedwebsocket "github.com/kubetail-org/kubetail/modules/shared/graphql/websocket"
```